lxd/storage/drivers/driver_powerflex.go

```diff
@@ -0,0 +1,208 @@
+package drivers
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/canonical/lxd/lxd/migration"
+	"github.com/canonical/lxd/lxd/operations"
+	"github.com/canonical/lxd/shared"
+	"github.com/canonical/lxd/shared/api"
+	"github.com/canonical/lxd/shared/validate"
+)
+
+// powerFlexDefaultUser represents the default PowerFlex user name.
+const powerFlexDefaultUser = "admin"
+
+// powerFlexDefaultSize represents the default PowerFlex volume size.
+const powerFlexDefaultSize = "8GiB"
+
+var powerFlexLoaded bool
+var powerFlexVersion string
+
+type powerflex struct {
+	common
+
+	client *powerFlexClient
```

Comment:
Can you add a comment here to indicate that one shouldnt use this var directly but instead call  `powerFlex()`

Reply:
Added.